portal/src/radix-theme-overrides.css

```diff
@@ -0,0 +1,18 @@
+.radix-themes {
+  --default-font-family: inherit;
```

Comment:
Do we still need to override `--default-font-family`?

Reply:
Yes, because by default all radix component uses the default value which is not same as what we specified in index.css. Change it to inherit so radix component also uses the same font-family of the whole portal.

Reply:
I see. BTW, we forgot to update font-family to follow the latest value we observed on github.com 

`-apple-system,BlinkMacSystemFont,"Segoe UI","Noto Sans",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji"`

Reply:
Added one more commit for this